### DISCUSSION TITLE
Recommended DB Size for AllReports

### DISCUSSION THREAD

**Discussion by successhawk** [vinijmoura/Azure-DevOps#20] [Q&A]
Hi, I'm looking to try out AllReports.  Can you make some recommendations of Azure SQL DB specs based on your experience?  
What's the least expensive config I can get away with?

**Comment by vinijmoura** [comment]
My size: Standard S0: 10 DTUs
My month cost: US$ 15.00

**Reply by successhawk** [reply]
Does the PAT really need to have full access? Shouldn't Analytics only need read only?  Unfortunately, admins in the org I'm in are going to push back.

**Reply by vinijmoura** [reply]
Sure! You need only a read access on all scopes :-)
Good suggestion here. I will change a documentation!
